src/dashboard.py

```diff
@@ -1761,6 +1760,7 @@ def API_AuthenticateLogin():
 def API_SignOut():
     resp = ResponseObject(True, "")
     resp.delete_cookie("authToken")
+    session.clear()
```

Comment:
I presume this clears the session.

Reply:
Yep, it clears the current session, not others ;)